spec/unit/parser/resource_spec.rb

```diff
@@ -283,44 +284,69 @@ class { alpha: stage => before }
       expect(edges).to include(['One[a]', 'Notify[bob says hello]'])
       expect(edges).to include(['One[b]', 'Notify[bill says hello]'])
     end
+
+    it 'should override default values with new defaults' do
```

Comment:
Technically the override is not a "new defaults" is it? It is just an override - an assignment.